.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore generated Maven build files
+**/target/
+
+**/walkthroughs/
+
+package-lock.json
+node_modules/
```

Comment:
Can you merge master into here? I think that should remove this file from the diff.

Reply:
Will pull upstream after step 5 merge.